src/cargo/core/compiler/fingerprint/mod.rs

```diff
@@ -376,6 +377,7 @@ use anyhow::format_err;
 use anyhow::Context as _;
 use cargo_util::paths;
 use filetime::FileTime;
+use lazycell::LazyCell;
```

Comment:
imo we should not be adding any new uses of lazycell but should instead be trying to remove it in favor of OnceCell